### ISSUE TITLE
0.13.0 Tooltip: Gatsby SSR Fails with "window not available during server side rendering"

### ISSUE THREAD

**Issue by karlhorky** [reach/reach-ui#738]
### Current Behavior

Running `gatsby build` on an app using `TooltipPopup` and `useTooltip` from `@reach/tooltip` fails:

```
$ cross-env NODE_ENV=production gatsby build

...

failed Building static HTML for pages - 0.750s
error "window" is not available during server side rendering.

  246 |   function wrapMouseEvent(theirHandler, ourHandler) {
  247 |     // Use internal MouseEvent handler only if PointerEvent is not supported
> 248 |     if ("PointerEvent" in window) return theirHandler;
      | ^
  249 |     return wrapEvent(theirHandler, ourHandler);
  250 |   }
  251 | 


  WebpackError: ReferenceError: window is not defined
  
  - tooltip.esm.js:248 
    [courses]/[@reach]/tooltip/dist/tooltip.esm.js:248:1
  
  - tooltip.esm.js:333 
    [courses]/[@reach]/tooltip/dist/tooltip.esm.js:333:1
  
  - index.js:30 
    reach-triangle-tooltip/index.js:30:40
  
  - objectWithoutProperties.js:2 
    [courses]/[@babel]/runtime/helpers/esm/objectWithoutProperties.js:2:16
  
  - getPrototypeOf.js:4 
    [courses]/[@babel]/runtime/helpers/getPrototypeOf.js:4:1
  

not finished Generating image thumbnails - 147.857s
```

The code in `reach-triangle-tooltip/index.js` is based on the demo code from [Triangle pointers and custom styles](https://reach.tech/tooltip/#triangle-pointers-and-custom-styles) from the docs (see component in sandbox below).

### Expected behavior

No error with server-side rendering

### Reproducible example

[CodeSandbox](https://codesandbox.io/s/epic-golick-12j0l?file=/src/pages/index.js)

### Your environment

| Software         | Name(s) | Version |
| ---------------- | ------- | ------- |
| Reach Package    |  tooltip, portal       |   0.13.0      |
| React            | react, react-dom        |  17.0.1       |
| Browser          |     macOS Chrome    |     latest    |
| Assistive tech   |  -       |         |
| Node             |    node     |   v15.5.0      |
| npm/yarn         |   yarn     |     v1.22.10    |
| Operating System |    macOS Big Sur     |   11.1      |

**Comment by karlhorky** [comment]
Here's the patch for anyone needing to fix this right away using [`patch-package`](https://www.npmjs.com/package/patch-package):

```diff
diff --git a/node_modules/@reach/tooltip/dist/tooltip.esm.js b/node_modules/@reach/tooltip/dist/tooltip.esm.js
index 94a8b4b..d52d26d 100644
--- a/node_modules/@reach/tooltip/dist/tooltip.esm.js
+++ b/node_modules/@reach/tooltip/dist/tooltip.esm.js
@@ -1,10 +1,10 @@

 function _extends() {
   _extends = Object.assign || function (target) {
@@ -245,7 +245,7 @@ function useTooltip(_temp) {
 
   function wrapMouseEvent(theirHandler, ourHandler) {
     // Use internal MouseEvent handler only if PointerEvent is not supported
-    if ("PointerEvent" in window) return theirHandler;
+    if (typeof window !== 'undefined' && "PointerEvent" in window) return theirHandler;
     return wrapEvent(theirHandler, ourHandler);
   }
 
```

**Comment by chaance** [comment]
Fixed in #742

**Comment by karlhorky** [comment]
Thanks @taifen and @chaance !
